docs/discord/discord-readme.md

```diff
@@ -93,18 +93,23 @@ Tips to help you ask questions that will give you a better chance to get the bes
 3. **Clearly describe the problem.** Explain what is happening and what you want to happen.
 4. **Share your code.** It’s very unlikely that anyone will be able to solve your problem without seeing your code. Share a GitHub repository, a GitHub gist link, or format the code by wrapping it in backticks, like the example below.
 
-\```c++
+````txt
+```ts
   // Your code goes here
-\``` <!-- In Discord, this will look fine -->
+```
+````
```

Comment:
A brief note that the nested codeblock is necessary to render this properly on the website, but when we pull these changes into discord we will need to remove the 4-tick level wrapper.

![image](https://user-images.githubusercontent.com/63889819/142970597-83c44a69-9b69-49c4-86eb-378656a358e3.png)